.github/workflows/haskell.yml

```diff
@@ -190,14 +201,12 @@ jobs:
         cache-name: cache-cabal-cabal-fmt
       with:
         path: ${{ steps.setup-haskell.outputs.cabal-store }}
-        key: ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}-${{ hashFiles('**/*.cabal') }}-${{ hashFiles('**/cabal.project') }}
-        restore-keys: |
-          ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}-${{ hashFiles('**/*.cabal') }}-${{ hashFiles('**/cabal.project') }}
-          ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}-${{ hashFiles('**/*.cabal') }}-
-          ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}-
+        key: ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}
 
     - name: Install cabal-fmt
-      run: cabal install cabal-fmt --constraint 'cabal-fmt == 0.1.10'
+      # --project-file="" ignores the cabal files in the current project. We
+      # don't need these packages to install cabal-fmt.
+      run: cabal install cabal-fmt --constraint 'cabal-fmt == 0.1.10' --project-file=""
```

Comment:
why not `--ignore-project` ?

Reply:
I thought the semantics of the two were slightly different, but having tried `--ignore-project` that also works